services/QuillLMS/spec/models/activity_session_spec.rb

```diff
@@ -829,16 +829,20 @@
     before { activity_session.update(visible: true) }
 
     it 'should create a concept result with the hash given' do
-      expect(OldConceptResult).to receive(:create).with({
-        activity_session_id: activity_session.id,
-        concept_id: concept.id,
-        metadata: metadata,
-        question_type: 'lessons-slide'
-      }).and_call_original
-      ActivitySession.save_concept_results([activity_session], concept_results)
+      Sidekiq::Testing.inline! do
+        expect do
+          expect(ConceptResult).to receive(:init_from_json).with({
```

Comment:
Is there a reason the spy `expect(ConceptResults)` is inside expect change matcher (instead of before it)?